src/common/k8s-api/api-manager.ts

```diff
@@ -11,16 +11,16 @@ import type { KubeApi } from "./kube-api";
 import type { KubeObject } from "./kube-object";
 import { IKubeObjectRef, parseKubeApi, createKubeApiURL } from "./kube-api-parse";
 
-export class ApiManager {
-  private apis = observable.map<string, KubeApi<KubeObject>>();
-  private stores = observable.map<string, KubeObjectStore<KubeObject>>();
+export class ApiManager<TKubeObject extends KubeObject = any> {
```

Comment:
Please no defaulting to `any`, or even better we should try and remove all uses of `any` since it basically is no better than JS.

Reply:
After discussing via Slack I completely agree, will change this PR to have proper types 👍

Reply:
Fixed 🎉